pkg/restclient/common.go

```diff
@@ -171,6 +209,32 @@ func (c *Client) do(method string, url string, input interface{}, expectStatusCo
 	return nil
 }
 
+func (c *Client) buildRequest(method string, url string, input interface{}) (*http.Request, error) {
+	url = fmt.Sprintf("%s%s", c.baseURL, url)
+	var body io.Reader
+	if input != nil {
+		reqBody, marshalErr := json.Marshal(input)
```

Comment:
use `err` instead of `marshalErr` as we are following `err` for error variable name.